src/App.js

```diff
@@ -1,14 +1,29 @@
 import React, { Component } from 'react';
+import thunkMiddleware from 'redux-thunk'
+import { createLogger } from 'redux-logger';
+import { Provider } from 'react-redux';
+import { createStore, applyMiddleware } from 'redux';
 import QuoteBoxContainer from './QuoteBoxContainer';
+import quoteReducer from './quoteReducer';
```

Comment:
Check if Your linter works, ./quoteReducer does not exist...should be ./QuoteReducer

Reply:
I'm installing extension.